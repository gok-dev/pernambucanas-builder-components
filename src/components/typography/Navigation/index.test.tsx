// import { renderWithTheme } from 'utils/helpers'
// import { Navigation } from './index'

// describe('<Navigation />', () => {
//   it('shold render the heading', () => {
//     const { container } = renderWithTheme(<Navigation />)
//     expect(container.firstChild).toMatchSnapshot()
//   })
// })
