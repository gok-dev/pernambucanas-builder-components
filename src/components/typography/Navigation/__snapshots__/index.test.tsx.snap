// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navigation /> shold render the heading 1`] = `
.c0 {
  font-weight: 400;
  font-size: 14px;
  font-weight: 300;
}

.c0 a {
  color: #2485EA;
  -webkit-text-decoration: none;
  text-decoration: none;
  -webkit-transition: all 0.3s;
  transition: all 0.3s;
}

.c0 a:hover {
  color: #1C67B8;
}

.c0 mark {
  border-radius: 3px;
  padding: 3px 4px;
}

<p
  class="c0"
  size="14"
/>
`;
